documents/readthedoc/docs/source/Solutions/tensorflow-serving-cluster/index.rst

```diff
@@ -181,29 +181,40 @@ The converted model file will be under::
 
    models/resnet50-v15-fp32/1/saved_model.pb
 
-1.2 Create the TLS certificate
+1.2 Create the SSL/TLS certificate
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-We choose gRPC TLS and create the one-way TLS Keys and certificates by setting
+We choose gRPC SSL/TLS and create the SSL/TLS Keys and certificates by setting
 TensorFlow Serving domain name to establish a communication link between client
 and TensorFlow Serving.
 
+For ensuring security of the data being transferred between a client and server, SSL/TLS can be implemented either one-way or two-way. 
```

Comment:
SSL/TLS can be implemented either one-way authentication or two-way authentication.

Reply:
> SSL/TLS can be implemented either one-way TLS authentication or two-way TLS authentication (mutual TLS authentication).

Updated.